Meadowlark-js/packages/meadowlark-core/src/handler/Delete.ts

```diff
@@ -27,13 +28,17 @@ export async function deleteIt(frontendRequest: FrontendRequest): Promise<Fronte
   try {
     writeRequestToLog(moduleName, frontendRequest, 'deleteIt');
 
-    if (frontendRequest.middleware.pathComponents.resourceId == null) {
+    if (frontendRequest.middleware.pathComponents.documentUuid == null) {
       writeDebugStatusToLog(moduleName, frontendRequest, 'deleteIt', 404);
       return { statusCode: 404 };
     }
-
+    const meadowlarkId = meadowlarkIdForDocumentInfo(
+      frontendRequest.middleware.resourceInfo,
+      frontendRequest.middleware.documentInfo,
+    );
     const request: DeleteRequest = {
-      id: frontendRequest.middleware.pathComponents.resourceId,
+      documentUuid: frontendRequest.middleware.pathComponents.documentUuid,
+      meadowlarkId,
```

Comment:
There is no body for a delete request, so there is no document to extract a DocumentIdentity from, so the meadowlarkId that can be computed is useless (only has the resource information). It should be removed from DeleteRequest.